doc/source/examples/similarity_explanations_20ng.ipynb

```diff
@@ -0,0 +1,1201 @@
+{
```

Comment:
<p>For some reason I prefer to use present simple ... feel free to ignore all the comments where I am asking you to change.</p><p> </p><p>Evaluating -&gt; evaluate</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>